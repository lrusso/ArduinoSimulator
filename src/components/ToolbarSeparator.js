import React from "react"

const ToolbarSeparator = () => {
  return <div style={styles.separator}></div>
}

const styles = {
  separator: {
    float: "left",
    borderLeft: "thin solid #D3D3D3",
    marginLeft: "5px",
    height: "100px",
    width: "1px",
  },
}

export default ToolbarSeparator
