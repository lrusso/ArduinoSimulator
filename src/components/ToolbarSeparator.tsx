import React from "react"

const ToolbarSeparator = () => {
  return <div style={styles.separator}></div>
}

const styles: { [key: string]: React.CSSProperties } = {
  separator: {
    float: "left",
    borderLeft: "2px solid #363636",
    marginLeft: "5px",
    height: "100px",
    width: "1px",
  },
}

export default ToolbarSeparator
