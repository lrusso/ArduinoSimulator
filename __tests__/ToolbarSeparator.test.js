import React from "react"
import renderer from "react-test-renderer"
import ToolbarSeparator from "../src/components/ToolbarSeparator"

describe("ToolbarSeparator", () => {
  test("Rendering 1", () => {
    const tree = renderer.create(<ToolbarSeparator />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
