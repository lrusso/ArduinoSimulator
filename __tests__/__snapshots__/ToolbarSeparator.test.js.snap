// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarSeparator Rendering 1 1`] = `
<div
  style={
    {
      "borderLeft": "thin solid #D3D3D3",
      "float": "left",
      "height": "100px",
      "marginLeft": "5px",
      "width": "1px",
    }
  }
/>
`;
